doc/source/user_guide/indexing.rst

```diff
@@ -933,6 +933,24 @@ and :ref:`Advanced Indexing <advanced>` you may select along more than one axis
 
    df2.loc[criterion & (df2['b'] == 'x'), 'b':'c']
 
+.. warning::
+
+   ``iloc`` supports two kinds of boolean indexing. If the indexer is a boolean ``Series``,
+   an error will be raised. For instance, in the following example, ``df.iloc[sr.values, 1]`` is ok.
```

Comment:
sr -> s

Reply:
Thanks for the quick review! I forgot to fix the doc. Done.